examples/open_value_sharing/test/test.ts

```diff
@@ -0,0 +1,14 @@
+import { getCanisterId } from 'azle/dfx';
+import { runTests } from 'azle/test';
+
+import { actorConsumer, agent } from './actor_consumer';
+import { createActor as createActorWallet } from './dfx_generated/wallet';
```

Comment:
how does createWalletActor and walletActor strike you?

Reply:
Changed